cmd/ak/cmd/vars/set.go

```diff
@@ -44,7 +47,7 @@ var setCmd = common.StandardCommand(&cobra.Command{
 			value = string(data)
 		}
 
-		ev := sdktypes.NewVar(n).SetValue(value).SetSecret(secret).SetOptional(optional).WithScopeID(id)
+		ev := sdktypes.NewVar(n).SetValue(value).SetSecret(secret).SetOptional(optional).WithScopeID(id).SetDescription(description)
```

Comment:
Using a builder-like pattern when you set all the values is an incorrect usage of the builder pattern. This was not added in the PR, but this PR makes it even clearer that it's not the right choice.